hub/policy/src/test/java/uk/gov/ida/hub/policy/domain/controller/IdpSelectedStateControllerTest.java

```diff
@@ -67,7 +47,7 @@
     private static final SessionId NEW_SESSION_ID = aSessionId().build();
     private static final List<LevelOfAssurance> levelsOfAssurance = asList(LevelOfAssurance.LEVEL_1, LevelOfAssurance.LEVEL_2);
     private static final LevelOfAssurance providedLevelOfAssurance = LevelOfAssurance.LEVEL_2;
-    private static final Optional<String> principalIpAddressAsSeenByIdp = Optional.fromNullable("principal-ip-address-from-idp");
+    private static final Optional<String> principalIpAddressAsSeenByIdp = Optional.of("principal-ip-address-from-idp");
```

Comment:
Convention is for static final things to be UPPER_CASE.
(we seem to not be doing that in some of this class though - would you mind fixing it?)